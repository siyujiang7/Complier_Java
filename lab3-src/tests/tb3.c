/*  Take CS636 for a good time :) */
struct ip6_hdr {
   union {
      struct {
         long ip6_un1_flow; /* 4 bits version, 8 bits TC, 20 bits
                                      flow-ID */
         long ip6_un1_plen; /* payload length */
         long ip6_un1_nxt; /* next header */
         uint8_t ip6_un1_hlim; /* hop limit */
      } ip6_un1;
      uint8_t ip6_un2_vfc; /* 4 bits version, top 4 bits
                                      tclass */
   } ip6_ctlun;
   struct in6_addr ip6_src; /* source address */
   struct in6_addr ip6_dst; /* destination address */
};
