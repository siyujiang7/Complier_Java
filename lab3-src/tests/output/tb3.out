struct ip6_hdr {
  union {
    struct {
      long ip6_un1_flow;
      long ip6_un1_plen;
      long ip6_un1_nxt;
      long ip6_un1_hlim;
    } ip6_un1;
    long ip6_un2_vfc;
  } ip6_ctlun;
  struct in6_addr ip6_src;
  struct in6_addr ip6_dst;
};
